sql/core/src/test/resources/sql-tests/inputs/tablesample-negative.sql

```diff
@@ -0,0 +1,13 @@
+-- Negative testcases for tablesample
+CREATE DATABASE mydb1;
+USE mydb1;
+CREATE TABLE t1 USING parquet AS SELECT 1 AS i1;
+
+-- Negative tests: negative percentage
+SELECT mydb1.t1 FROM t1 TABLESAMPLE (-1 PERCENT);
```

Comment:
Add a comment in the test case to explain the SQL API always assume `withReplacement` is set to `false`